get rid of duplicate tables (#3529)
segmentio/segment-docs: src/connections/destinations/catalog/actions-intercom-cloud/index.md

{% include components/actions-fields.html settings="true"%}

## FAQ & Troubleshooting

### Why is a company I created missing from my Intercom dashboard?

> If a company is created without an attached user, the company will not appear on Intercom's dashboard. This is expected. Once a user is attached to the company, it should appear in the list of companies.

Reviewer: ```suggestion
If a company is created without an attached user, the company does not appear on Intercom's dashboard. This is expected. Once a user is attached to the company, it should appear in the list of companies.
```

### Why isn’t a user getting attached to a company?

> When using the Identify Company action, Segment will first create or update a company. In the same action, Segment will also attach the user in your group call to that company. If the user does not exist in Intercom when the action runs, we will create/update the company but not attach the user. Please ensure the user is created in Intercom first.

Reviewer: ```suggestion
When you use the Identify Company action, Segment creates or updates a company's information. In the same action, Segment also attaches the user in your group call to that company. If the user doesn't exist in Intercom when the action runs, Segment creates or updates the company but not attach the user. Ensure the user is created in Intercom first.
```
